docs/agent-reference/how-to-run-agent.md

```diff
@@ -8,12 +8,24 @@ AducIotAgent '<IoT device connection string>' [options...]
 
 ### Additional Arguments
 
-#### --EnableIotHubTracing
+#### --version
+
+With this option enabled, the reference agent will output the version.
+
+#### --enable-iothub-tracing
 
 With this option enabled, the reference agent will output verbose tracing from
 the Azure IoT C SDK. This option is useful for troubleshooting connection
 issues.
 
+#### --health-check
+
+With this option enabled, the reference agent will turn on Health Check Only mode.
```

Comment:
unclear how this description is helpful, or if health-check actually adds any value curently.  I've mentioned as such to Nox.